code/clue.js

```diff
@@ -1,59 +1,361 @@
+  favoriteWeapon: ''
+};
+
+const mrsWhite = {
+  firstName: 'Wendy',
+  lastName: 'White',
+  color: '#ebebe9',
+  description: 'A sweet old lady',
+  age: 73,
+  image: 'assets/white.png',
+  occupation: 'Servant',
+  favoriteWeapon: ''
+};
 
 const rope = {
-  name: 'rope',
-  weight: 10
-}
+  name: 'Rope',
+  id: 'rope',
+  weight: 10,
+  color: '#f84b3e',
+  article: 'a',
+  increasedChanceOfDraw: 6,
+  likelyFoundIn: 'Guest House'
+};
+
+const knife = {
+  name: 'Knife',
+  id: 'knife',
+  weight: 7,
+  color: '#fff',
+  article: 'a',
+  increasedChanceOfDraw: 5,
+  likelyFoundIn: 'Kitchen'
+};
+
+const candleStick = {
+  name: 'Candle stick',
+  id: 'candleStick',
+  weight: 4,
+  color: '#8b5195',
+  article: 'a',
+  increasedChanceOfDraw: 10,
+  likelyFoundIn: 'Hall'
+};
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const dumbBell = {
+  name: 'Dumbbell',
+  id: 'dumbBell',
+  weight: 12,
+  color: '#ebebe9',
+  article: 'a',
+  increasedChanceOfDraw: 9,
+  likelyFoundIn: 'Spa'
+};
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+const poison = {
+  name: 'Poison',
+  id: 'poison',
+  weight: 3,
+  color: '#5656b3',
+  article: '',
+  increasedChanceOfDraw: 7,
+  likelyFoundIn: 'Study'
+};
+
+const axe = {
+  name: 'Axe',
+  id: 'axe',
+  weight: 6,
+  color: '#fff',
+  article: 'an',
+  increasedChanceOfDraw: 10,
+  likelyFoundIn: 'Billiard Room'
+};
+
+const bat = {
+  name: 'Bat',
+  id: 'bat',
+  weight: 2,
+  color: '#fff',
+  article: 'a',
+  increasedChanceOfDraw: 20,
+  likelyFoundIn: 'Lounge'
+};
+
+const trophy = {
+  name: 'Trophy',
+  id: 'trophy',
+  weight: 2,
+  color: '#539853',
+  article: 'a',
+  increasedChanceOfDraw: 8,
+  likelyFoundIn: 'Theater'
+};
+
+const pistol = {
+  name: 'Pistol',
+  id: 'pistol',
+  weight: 2,
+  color: '#fbd460',
+  article: 'a',
+  increasedChanceOfDraw: 25,
+  likelyFoundIn: 'Living Room'
+};
 
 const suspects = [
   mrGreen,
+  clMustard,
+  mrsPeacock,
+  prPlum,
+  msScarlet,
   mrsWhite
-  // ...  and the rest
 ]
 
-const weapons = []
+const weapons = [
+  rope,
+  knife,
+  candleStick,
+  dumbBell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+]
 
-const rooms = []
+const rooms = [
+  'Dining Room',
+  'Conservatory',
+  'Kitchen',
+  'Study',
+  'Library',
+  'Billiard Room',
+  'Lounge',
+  'Ballroom',
+  'Hall',
+  'Spa',
+  'Living Room',
+  'Observatory',
+  'Theater',
+  'Guest House',
+  'Patio'
+];
 
-// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
 }
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
+const mystery = {
+  killer: null,
+  weapon: null,
+  room: null
+};
+
+const pickKiller = (loadID) => {
+  if (gameStarted && !killerPicked) {
```

Comment:
One idea here could be to 
```Javascript
if( !gameStarted || killerPicked) {
   return;
}

// Now you know the game is started and the killer is not picked
```

Inverting your logic would allow you to not have the entire function in the `if` 👍